src/gl.rs

```diff
@@ -1112,6 +1112,24 @@ pub fn get_program_iv(program: GLuint, pname: GLenum) -> GLint {
 }
 
 #[inline]
+pub fn get_vertex_attrib_iv(index: GLuint, pname: GLenum) -> GLint {
+    unsafe {
+        let mut result: GLint = 0 as GLint;
+        ffi::GetVertexAttribiv(index, pname, &mut result);
+        return result;
+    }
+}
+
+#[inline]
+pub fn get_vertex_attrib_fv(index: GLuint, pname: GLenum) -> Vec<GLfloat> {
+    unsafe {
+        let mut result = vec![0 as GLfloat; 4];
```

Comment:
something about this function isn't right. I'm getting this crash:
```
 │ thread 'WebGLThread' panicked at 'Unexpected WebGL error: 0x502 (1282)', /Users/dzbarsky/mozilla/webrender_traits/src/webgl.rs:442
  │ stack backtrace:
  │    1:        0x10a94f4f8 - std::sys::backtrace::tracing::imp::write::h714760a4c8c0cdd8
  │ FAIL gl.getVertexAttrib(0, gl.CURRENT_VERTEX_ATTRIB)[0] should be 1. Was 0.
  │    2:        0x10a9557d5 - std::panicking::default_hook::_$u7b$$u7b$closure$u7d$$u7d$::hff309ab1d83ffd90
  │    3:        0x10a9553ee - std::panicking::default_hook::h08ad3bb09872855b
  │    4:        0x10a0dfc2e - util::panicking::initiate_panic_hook::_$u7b$$u7b$closure$u7d$$u7d$::_$u7b$$u7b$closure$u7d$$u7d$::he7fa94d82e246627
  │    5:        0x10a93d0b8 - std::sys_common::unwind::begin_unwind_inner::h406d5f1a330b854b
  │    6:        0x10a93e9ee - std::sys_common::unwind::begin_unwind_fmt::h57ea3fbee1a40196
  │    7:        0x1095b2791 - webrender_traits::webgl::WebGLCommand::apply::hf26127a7e3bda5c2
  │    8:        0x1095ae7e2 - canvas::webgl_paint_thread::WebGLPaintThread::handle_webgl_message::h27afef3930098b0d
```

from this: 
```
gl.vertexAttrib1fv(ii, [1]);
shouldBe('gl.getVertexAttrib(' + ii + ', gl.CURRENT_VERTEX_ATTRIB)[0]', '1');
``

If I take out the `ffi:GetVertexAttribfv` call and return all 0's, no crash